src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,42 +29,43 @@ export function getStaticProps(context) {
 
 }
 
-**In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-In the launch of public preview, Amplify code-first DX (Gen 2) supports Auth and Data. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs.
+**Is Gen 2 backward compatible with existing Amplify projects on Gen 1?**
 
-**Does Amplify Gen 2 support DataStore?**
+No, Gen 2 is not backward compatible with existing Amplify projects on Gen 1. Amplify Gen 1 and Gen 2 follow different architectural and tooling paradigms in order for Gen 2 to address common feedback customers had about Gen 1. The two versions are not compatible.
 
-No, in public preview, Amplify code-first DX (Gen 2) supports GraphQL APIs without DataStore.
+**Is there a way to upgrade an existing Amplify project from Gen 1 to Gen 2?**
```

Comment:
I'd actually put this as the first question

Reply:
done